### DISCUSSION TITLE
Reports subtitles

### DISCUSSION THREAD

**Discussion by sevenfifty777** [rkusa/DATIS#126] [Q&A] [answered]
Hi rkusa
not sure it is right place for this question
is there a way to get subtitles of reports on the game screen during radio communication?

**Comment by rkusa** [comment, marked as answer]
Hi @sevenfifty777, not right now I am afraid. But it might be possible in future with changes that are already ongoing in the background.

**Reply by sevenfifty777** [reply]
Thx for the info

**Comment by chipwich** [comment]
I kind of like it without subtitles, but repeating as it does in real life ATIS.
